Remotion/Data/DomainObjects.UnitTests/Mapping/RelationDefinitionChecker.cs

```diff
@@ -69,7 +69,7 @@ public void Check (RelationDefinition expectedDefinition, RelationDefinition act
       CheckEndPointDefinitions(expectedDefinition, actualDefinition);
     }
 
-    private void CheckEndPointDefinitions (RelationDefinition expectedRelationDefinition, RelationDefinition actualRelationDefinition) // TODO R2I this needs to be rewritten for TypeDefinition
+    private void CheckEndPointDefinitions (RelationDefinition expectedRelationDefinition, RelationDefinition actualRelationDefinition)
```

Comment:
what has changed that no rewrite/extension is needed?

Reply:
The todo was just stale and the code part has already been changed